ecms_base/themes/custom/ecms/ecms.theme

```diff
@@ -523,6 +523,7 @@ function ecms_preprocess_paragraph__file_list_by_tag(array &$variables): void {
   $query->condition('bundle', 'file')
     ->condition('field_file_tags', $term_id)
     ->condition('status', 1)
+    ->sort('field_file_list_weight', "ASC")
```

Comment:
What happens to existing entities that don't have a value for this yet - does the query still work?

Reply:
Hmm, I cant really test that at this point. However, we did do this same thing with person nodes that already existed and it worked fine. My guess is they are null in the database and so it treats it if you were calling a sort on items with values as well as null.

Reply:
Ok, we can test once this hits stage. I will setup a site now with a test paragraph.